RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTHcalVarProducerFromRecHit.cc

```diff
@@ -229,12 +229,12 @@ void EgammaHLTHcalVarProducerFromRecHit::produce(edm::StreamID,
       int iEA = -1;
       auto scEta = std::abs(recoEcalCandRef->superCluster()->eta());
       for (int bIt = absEtaLowEdges_.size() - 1; bIt > -1; bIt--) {
-        if (scEta > absEtaLowEdges_.at(bIt)) {
+        if (scEta >= absEtaLowEdges_[bIt]) {
```

Comment:
most probably  
```int iEA = std::lower_bound(absEtaLowEdges_.begin(),absEtaLowEdges_.end(),scEta)```
would do it better.
(why we need always to re-invent the way to find a bin in a Histogram?)

Reply:
I considered that, this is the minimal fix.